include/libdnf5/conf/config.hpp

```diff
@@ -33,8 +33,9 @@ namespace libdnf5 {
 /// Base class for configurations objects
 class Config {
 public:
+#ifndef SWIGGO
```

Comment:
I would rather keep these`SWIGGO` conditionals only in the interface files.
I think it could look something like:
```
#if defined(SWIGGO)
%ignore libdnf5::Config::opt_binds;
#endif
```